### PR TITLE
Fix simple-analysis link

### PR DESCRIPTION
**Pull request by mxxo**
Hi! This is a very cool collection of crates :) 
I found this broken link to `simple-analysis`.

### CODE REVIEW

File: malice/README.md [hunk 1]
```diff
@@ -8,8 +8,7 @@ https://docs.rs/malice/
 
 Here is a very simple example analysis using `malice` and other crates from this repository.
 It measures the pseudorapidity distribution of the reconstructed tracks.
-For a more comprehensive, but still small, example (including concurrency) check out [simple-analysis](https://github.com/cbourjau/alice-rs/tree/master/simple-analysis).
-
+For a more comprehensive, but still small, example (including concurrency) check out [simple-analysis](!ttps://github.com/cbourjau/alice-rs/tree/master/examples/simple-analysis).
```
Comment (cbourjau) [inline malice/README.md:11]: I think there might be a typo here unless thats some markdown magic (`!ttps`)

Comment (mxxo) [reply malice/README.md]: Whoops, sorry about that.

**Comment by mxxo** [comment]
Good catch, thanks :)

**Comment by cbourjau** [comment]
Thanks again! Reminds me that I ought to fix the CI :D

**Pull request merged**